includes/integration.drush.inc

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @file
+ * Contains Drush-related hook implementations.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function integration_drush_command() {
+  $items = array();
+  $items['integration-export-document'] = array(
```

Comment:
I add 'document' since we 'export' only one 'document', while integration-export-all (for instance) would export all 'documents'. Or we can use 'integration-export' for both cases, but we will need more options and code, while I wanted the code to be very simple, to start with.